about.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>KELSEY MCALPINE | About</title>
+    <link rel="stylesheet" href="styles/normalize.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles/styles.css">
+  </head>
+  <body id="about">
+    <header>
+      <section id="header-top">
+        <h5 class="left">FULL-STACK DEVELOPER | SEATTLE, WA</h5>
+        <ul class="right">
+          <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+          <li><a href="#"><i class="fa fa-linkedin"></i></a></li>
+          <li><a href="#"><i class="fa fa-github"></i></a></li>
+        </ul>
+      </section>
+      <section id="header-bottom">
+        <h1 class="left">KELSEY MCALPINE</h1>
+        <ul class="right">
+          <li><h3><a href="blog.html">Blog</a></h3></li>
+          <li><h3><a href="about.html">About</a></h3></li>
+          <li><h3><a href="index.html">Work</a></h3></li>
+        </ul>
+      </section>
+    </header>
+    <main>
+      <section class="container">
+        <img src="http://lorempixel.com/g/400/500" alt="placeholder">
+        <section id="about-info">
```

Comment:
This section changes its position with different screen resolutions. If you make the screen smaller, it drops to the bottom of the image. No big deal. You will learn how to use Foundation and/or Bootstrap to make your screen more responsive later.